### DISCUSSION TITLE
vue/quasar/tailwind update process

### DISCUSSION THREAD

**Discussion by misterkramer** [zauberzeug/nicegui#318] [Ideas / Feature Requests]
Hi! 
I would like to know if there will be synchronization process with `vue3/quasar/tailwind` releases in order to always use the latest stable versions in [static](https://github.com/zauberzeug/nicegui/tree/main/nicegui/static)? 
I want to avoid "problem" like in JustPy with the lag of `vue/quasar` versions. Now there is already a small lag in the versions of `vue/quasar`, for example quasar latest stable version is [2.11.5](https://quasar.dev/start/release-notes), in `nicegui` - [2.10.0](https://github.com/zauberzeug/nicegui/blob/main/nicegui/static/quasar.umd.prod.js#L2)

**Comment by falkoschindler** [comment]
Thanks, @misterkramer, for bringing this up! This is a very interesting point.

We could update dependencies for every minor version upgrade, e.g. from NiceGUI 1.1.x to 1.2.0. This way we shouldn't deviate too much from the latest versions. What do you think?

**Reply by misterkramer** [reply]
Yes, this approach looks good to start with. In the future, we can think about autoPR with bump deps - it will be necessary to track new versions automatically (I think it can be implemented via github actions).

**Comment by falkoschindler** [comment]
For version 1.2.0 we updated all dependencies. And we're planning on doing so with every minor version upgrade.
Therefore, I'll close this discussion as completed. ✔️
